electron/home.js

```diff
@@ -45,10 +46,42 @@ window.onload = function () {
                     option.innerHTML = device.label;
                     option.value = device.deviceId;
                     audioInputSelect.appendChild(option);
-                    console.log(device.label);
                 }
             });
         })
+
+    watchForAndDisplayResult();
+}
+
+async function watchForAndDisplayResult() {
+    fs.watch(result_path, (eventType, filename) => {
```

Comment:
really neat way of checking for the result. we'll need to be VERY careful that nothing else changes the txt file since that triggers the front end adjustments. are we going to continue with this route or do we plan on doing message passing between python/js?

if we go this route, we'll probably need the classifier to export the result to two different txt files: one being just the result.txt which is parsed by javascript, and the other that serves an archival purpose (e.g., <yyyy-MM-ddTHH:mm:ss>.txt) which will also contain classification times/statistics.

Reply:
"if we go this route, we'll probably need the classifier to export the result to two different txt files: one being just the result.txt which is parsed by javascript, and the other that serves an archival purpose (e.g., yyyy-MM-ddTHH:mm:ss.txt) which will also contain classification times/statistics." -- exactly what I was thinking.

Writing and reading to a textfile seems a bit bush league but I think it is fine. If we start doing a lot more message passing between electron and python this solution will not scale well, but I don't foresee us needing more messaging.